.github/workflows/gh-ci.yml

```diff
@@ -13,6 +13,20 @@ name: VeniceCI
 on: [push, pull_request, workflow_dispatch]
 
 jobs:
+  ValidateGradleWrapper:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 5
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Validate Gradle wrapper
+        uses: gradle/wrapper-validation-action@v1
+
   StaticAnalysis:
```

Comment:
> Should we combine static analysis + gradle validation in a single job?

I prefer to leave them separate for now since StaticAnalysis is a Gradle execution and ValidateGradleWrapper has no Gradle execution stuff. The implementation of adding the gradle wrapper in that job would require a bit of string manipulation magic, and I don't think we really need that. What do you think?

Reply:
I was thinking instead of `.github/rawWorkflows/gh-ci-validate-gradle-wrapper.txt` we can have `.github/rawWorkflows/gh-ci-validate-gradle-wrapper-and-static-analysis.yml` for static analysis + gradlew validation. That way we can completely get rid of trickery for static analysis and validation jobs from `build.gradle`.

Reply:
I still like the current approach since the one you mentioned would require us to maintain almost duplicate jobs - java version, gradle args, artifact packaging and uploading in separate files - where the parametrized one can work well for `StaticAnalysis` in it's current state.

In the current approach, I think we never need to change `gh-ci-validate-gradle-wrapper.txt` since it is quite minimal already

Reply:
Ok. Sounds good.